synapseformation/schemas/long.schema.yml

```diff
@@ -0,0 +1,155 @@
+$base: "https://synapse.org#"
+
+$namespaces:
+  syn: "https://synapse.org#"
+
+$graph:
+
+- name: RootDoc
+  type: documentation
+  doc:
+    - |
+      # SynapseFormation Long Template
+
+      TBD
+      
+
+- name: ResourceRecord
+  doc: |
+    The abstract base type for a Synapse resource.
+  type: record
+  abstract: true
+  fields:
+  - name: name
+    doc: ""
+    type: string?
+  - name: id
+    doc: "Synapse ID"
+    type: string?
+
+- name: EntityRecord
+  doc: |
+    The abstract base type for a Synapse entity.
+  type: record
+  extends: ResourceRecord
+  abstract: true
+  fields:
+  - name: children
+    doc: ""
+    type: FolderRecord[]?
```

Comment:
Would this be an issue once we start supporting Files?  I guess the other question is, will we even support files.  I can imagine this sort of scenario:

1. Create Synapse formation template
2. Run `synapseformation create`
3. Create manifest file with synapse ids from step 2
4. Run `synapseutils.syncToSynapse

Reply:
I think we should support files in a future version. We can tweak the schema when we update the test/example template in the same PR.